panther_battery/README.md

```diff
@@ -1,64 +1,39 @@
-[//]: # (ROS_API_PACKAGE_START)
-[//]: # (ROS_API_PACKAGE_NAME_START)
-
 # panther_battery
 
-[//]: # (ROS_API_PACKAGE_NAME_END)
-[//]: # (ROS_API_PACKAGE_DESCRIPTION_START)
-
-Package containing nodes monitoring and publishing the internal battery state of the Husarion Panther robot.
-
-[//]: # (ROS_API_PACKAGE_DESCRIPTION_END)
+The package containing nodes monitoring and publishing the internal battery state of the Husarion Panther robot.
 
-## ROS Nodes
+## Launch Files
 
-[//]: # (ROS_API_NODE_START)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_0)
-[//]: # (ROS_API_NODE_COMPATIBLE_1_2)
-[//]: # (ROS_API_NODE_NAME_START)
+This package contains:
 
-### battery_driver
+- `battery.launch.py`: Responsible for activating battery node, which dealing with reading and publishing battery data.
 
-[//]: # (ROS_API_NODE_NAME_END)
-[//]: # (ROS_API_NODE_DESCRIPTION_START)
+## ROS Nodes
 
-Publishes battery state read from ADC unit for Panther version 1.2 and above, or based on Roboteq motor controllers' data for earlier versions of the robot.
+- [`battery_node`](#battery_node): Publishes battery state read from ADC unit for Panther version 1.2 and above, or based on Roboteq motor controllers' data for earlier versions of the robot.
 
-[//]: # (ROS_API_NODE_DESCRIPTION_END)
+### battery_node
 
 #### Publishes
```

Comment:
I think we can now move description directly under node description as most packages have just 1-2 nodes.
It would look smt like:
```
## ROS Nodes

### node_name

Very interesting description of the node.

#### Publishes
```